helm/vitess/templates/_vttablet.tpl

```diff
@@ -109,6 +110,25 @@ spec:
       spec:
 {{ toYaml (.dataVolumeClaimSpec | default $defaultVttablet.dataVolumeClaimSpec) | indent 8 }}
 
+---
+###################################
+# vttablet PodDisruptionBudget
+###################################
+apiVersion: policy/v1beta1
+kind: PodDisruptionBudget
+metadata:
+  name: {{ $setName }}
```

Comment:
It's a bit overly defensive for an object name, but by convention we try to use ` | quote` for all strings.